src/components/kroniskkrav/KroniskKrav.test.tsx

```diff
@@ -12,6 +12,7 @@ import testOrganisasjon from '../../mockData/testOrganisasjoner';
 import LocaleProvider from '../../locale/LocaleProvider';
 
 const arbeidsgivere: Organisasjon[] = testOrganisasjon;
+jest.unmock('react-i18next');
```

Comment:
Går det an å lage mockOnce?

Reply:
jest.dontMock(moduleName) skal bare slå av for en enkelt test. unmock skal ta hele fila. Forutsatt at jeg har forstått dokumentasjonen riktig.